cpp/include/raft/neighbors/detail/cagra/compute_distance_vpq.cuh

```diff
@@ -88,6 +88,26 @@ struct cagra_q_dataset_descriptor_t : public dataset_descriptor_base_t<half, DIS
   {
   }
 
+  template <uint32_t DATASET_BLOCK_DIM>
+  __device__ void copy_query(const DATA_T* const dmem_query_ptr,
+                             QUERY_T* const smem_query_ptr,
+                             const std::uint32_t query_smem_buffer_length)
+  {
+    for (unsigned i = threadIdx.x * 2; i < dim; i += blockDim.x * 2) {
+      half2 buf2 = {CUDART_ZERO_FP16, CUDART_ZERO_FP16};
```

Comment:
I believe `CUDART_ZERO_FP16` is not available prior to cuda 12.2 and this will break CI.